.github/workflows/CD.yml

```diff
@@ -40,9 +64,9 @@ jobs:
           aws s3 cp $deb_artifact s3://artifacts.opendistroforelasticsearch.amazon.com/downloads/debs/opendistro-knn/
           aws cloudfront create-invalidation --distribution-id E1VG5HMIWI4SA2 --paths "/*"
 
-  library-build-and-ship-artifacts:
-    name: Build and release JNI library artifacts
-    runs-on: ubuntu-latest
+  library-build-and-ship-artifacts-x64:
```

Comment:
Out of curiosity, why is it x64 instead of x86?

Reply:
This is based on the new naming convention, x86 is 32bit, x64 is 64bit